op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -104,29 +75,36 @@ func applyStateMigrationChanges(genesis *core.Genesis, dbPath string, commit boo
 		return nil, err
 	}
 
-	// Create the header for the Bedrock transition block.
+	// Apply the changes to the state DB.
+	err = applyAllocsToState(db, genesis)
```

Comment:
Why have we moved this logic after the "Committing state DB" step?

Reply:
Nice catch, that's unintentional.